### PR TITLE
Minor changes to logging tab. More information and one correction

### PR DESCRIPTION
**Pull request by HThuren**
When using the loggin tab in betaflight, you have to no information before you do the selected logging. Added som leading text.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: src/tabs/logging.html
Comment (haslinghuis) [inline src/tabs/logging.html:39]:
```suggestion
                <dd>8 columns (Mot, Mot2, Mot3, Mot4, Mot5, Mot6, Mot7, Mot8)</dd>
```

Comment (HThuren) [reply src/tabs/logging.html]: Good idea, since this are my first PR, I hope I did it right, I presse resolved to agree the change.

Comment (haslinghuis) [reply src/tabs/logging.html]: Sorry. my wrong it should be `Mot1`

Also please try squashing commits using https://github.com/betaflight/betaflight/wiki/Betaflight-Contribution
Amend also works great with force push for minor updates.

Comment (HThuren) [reply src/tabs/logging.html]: Right Mo1 gives the right sequence

File: src/tabs/logging.html
Comment (haslinghuis) [inline src/tabs/logging.html:39]: DIdn't work please try

```suggestion
                <dd>8 columns (Mot1, Mot2, Mot3, Mot4, Mot5, Mot6, Mot7, Mot8)</dd>
```

**Review by haslinghuis — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Review by KarateBrot — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
